emails/2024-10-18.md

```diff
@@ -0,0 +1,27 @@
+
+It's spooky season here in Los Angeles. For those of you not from here, this city takes this holiday very seriously. I think it's the entertainment roots. People decorate more for Halloween than Christmas. I'm not a fan. I'm kind of the Halloween version of The Grinch.
+
+### Dolt Use Cases
+
+Every two years or so, we update our summary of [Dolt Use Cases](https://www.dolthub.com/blog/2024-10-15-dolt-use-cases/). We link to this article in the [Dolt README](https://github.com/dolthub/dolt?tab=readme-ov-file#whats-it-for) so it gets a fair amount of attention. If you're looking for inspiration on how to use Dolt, [this article](https://www.dolthub.com/blog/2024-10-15-dolt-use-cases/) is a good place to start. 
+
+If you're using Dolt in a way that isn't mentioned, we'd love to write about it and share it with the world. Just reply to this email and tell me more. I promise it will make my day.
+
+### Dolt Workbench App
+
+Previously, [the Dolt Workbench was available to deploy on any host via Docker](https://www.dolthub.com/blog/2023-11-29-dolt-workbench/). But let's be honest, SQL workbenches are not for infrastructure folks, the core Docker user base. SQL workbenches are for data analysts, data hobbyists, Excel junkies, and people who just **need** a GUI to get anything done. The Dolt Workbench needed to be a standalone desktop application. Download it, install it, and run it. No Docker required. 
+
+Well, [here we are](https://www.dolthub.com/blog/2024-10-16-dolt-workbench-electron-app/). The Dolt Workbench is available for [Mac](https://apps.apple.com/us/app/dolt-workbench/id6720702995?mt=12) and [Windows](https://github.com/dolthub/dolt-workbench/releases/download/v0.2.13/dolt-workbench-win-x64.exe) as a standalone application. We achieved this through [the power of Electron](https://www.dolthub.com/blog/2024-09-11-building-an-electron-app-with-nextjs/). 
+
+I've been using the Dolt Workbench instead of the command line shell and I really enjoy the experience. It's like Datagrip and Git had a baby and matured through 20 years of GUI development. We also just [announced Workbench support for Doltgres]() so use it with either.
+
+### Is MySQL Dying?
+
+I'm not sure if many of you saw but the commercial arm of MariaDB was [acquired at a massive discount](https://techcrunch.com/2024/09/10/mariadb-goes-private-with-new-ceo-as-k1-closes-acquisition/) to their initial offering price two years ago. This prompted me to write an [Is MySQL Dying? article](https://www.dolthub.com/blog/2024-10-14-is-mysql-dying/). 
+
+I'm of the opinion that AWS is draining all the profits from the MySQL ecosystem and this makes Oracle mad. And frankly, I don't blame Oracle. But, it's hurting the ecosystem, potentially mortally. Good thing [Doltgres is almost Beta](https://www.dolthub.com/blog/2024-08-06-doltgres-beta/). Don't worry. We'll continue to Stan for MySQL as long as we have people using Dolt.
```

Comment:
```suggestion
I'm of the opinion that AWS is draining all the profits from the MySQL ecosystem and this makes Oracle mad. And frankly, I don't blame Oracle. But, it's hurting the ecosystem, potentially mortally. Good thing [Doltgres is almost Beta](https://www.dolthub.com/blog/2024-08-06-doltgres-beta/). Don't worry. We'll continue to stand for MySQL as long as we have people using Dolt.
```

Reply:
Stan is slang Neil. 

https://www.merriam-webster.com/dictionary/stan

There's a whole Dido + Eminem song.